site.properties

```diff
@@ -17,7 +17,7 @@ site.filestore     = filestore
 site.pql           = pql
 site.logvisualizer = logvisualizer
 
-site.fullProtocolHostPortUrl=http://localhost:8181/
+site.fullProtocolHostPortUrl=http://apromorewebapp.apromoresso.net:8181/
```

Comment:
Presumably revealing this DNS name isn't a huge issue since it's not a publicly resolvable one, but it does scent slightly of leaking internal security details.

Reply:
Yep, it's actually important because otherwise one docker container can't find the other - but, we can fine tune later I think.